src/Algebra/Module/Morphism/Linear/Properties.agda

```diff
@@ -0,0 +1,127 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of linear maps.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Algebra                   using (CommutativeRing)
+open import Algebra.Module            using (Module)
+open import Level                     using (Level)
+
+module Algebra.Module.Morphism.Linear.Properties
```

Comment:
Finally by library convention, this should be called `Algebra.Module.Morphism.ModuleHomomorphism`. I know it has special significance as linear maps, but I think it is better to put comments to that effect both on the main record definition, and in the header of this module?